pom.client.xml

```diff
@@ -684,31 +712,69 @@
       </reporting>
     </profile>
 
-    <!-- Setup for Java 9+ -->
+    <!-- Setup for Java 11+ -->
     <profile>
       <id>java-lts</id>
       <activation>
-        <jdk>[9,)</jdk>
+        <jdk>[11,)</jdk>
       </activation>
       <build>
         <plugins>
           <plugin>
             <groupId>org.apache.maven.plugins</groupId>
             <artifactId>maven-compiler-plugin</artifactId>
             <version>${maven-compiler-plugin.version}</version>
-            <!-- setup defaults for compile and testCompile -->
             <configuration>
-              <source>9</source>
```

Comment:
Does this mean you we plan to still use the default target of java 8 for the compile and target now?

Reply:
The intention is to compile with JDK 11 but run to target JDK 8 APIs. There is a secondary run (internally) that compiles just the module-info.java files at JDK 11 level. These are then combined into a single release jar. This means that we can drop the JDK 8 build pipeline (we still keep JDK 8 for the test pipelines though).

Reply:
Sounds good.